idea-plugin/src/main/kotlin/io/github/composegears/valkyrie/ui/screen/mode/iconpack/creation/common/packedit/model/PackEditState.kt

```diff
@@ -0,0 +1,39 @@
+package io.github.composegears.valkyrie.ui.screen.mode.iconpack.creation.common.packedit.model
+
+import io.github.composegears.valkyrie.ui.domain.validation.InputState
+import io.github.composegears.valkyrie.ui.domain.validation.ValidationResult
+
+data class PackEditState(
+    val inputFieldState: InputFieldState = InputFieldState(
+        iconPackName = InputState(),
+        packageName = InputState(),
+        nestedPacks = emptyList(),
+    ),
+)
+
+data class InputFieldState(
+    val iconPackName: InputState,
+    val packageName: InputState,
+    val nestedPacks: List<NestedPack>,
+) {
+    val isValid: Boolean
+        get() = iconPackName.isValid() &&
```

Comment:
Seems this could be a property instead of a getter.

Reply:
yes)